--- b/gcc/config/xtensa/xtensa.c
+++ a/gcc/config/xtensa/xtensa.c
@@ -1773,7 +1773,7 @@
   rtx tgt = operands[callop];
 
   if (GET_CODE (tgt) == CONST_INT)
+    sprintf (result, "call8\t0x%lx", INTVAL (tgt));
-    sprintf (result, "call8\t" HOST_WIDE_INT_PRINT_HEX, INTVAL (tgt));
   else if (register_operand (tgt, VOIDmode))
     sprintf (result, "callx8\t%%%d", callop);
   else
@@ -2348,14 +2348,14 @@
 
     case 'L':
       if (GET_CODE (x) == CONST_INT)
+	fprintf (file, "%ld", (32 - INTVAL (x)) & 0x1f);
-	fprintf (file, HOST_WIDE_INT_PRINT_DEC, (32 - INTVAL (x)) & 0x1f);
       else
 	output_operand_lossage ("invalid %%L value");
       break;
 
     case 'R':
       if (GET_CODE (x) == CONST_INT)
+	fprintf (file, "%ld", INTVAL (x) & 0x1f);
-	fprintf (file, HOST_WIDE_INT_PRINT_DEC, INTVAL (x) & 0x1f);
       else
 	output_operand_lossage ("invalid %%R value");
       break;
@@ -2369,7 +2369,7 @@
 
     case 'd':
       if (GET_CODE (x) == CONST_INT)
+	fprintf (file, "%ld", INTVAL (x));
-	fprintf (file, HOST_WIDE_INT_PRINT_DEC, INTVAL (x));
       else
 	output_operand_lossage ("invalid %%d value");
       break;
@@ -2438,7 +2438,7 @@
       else if (GET_CODE (x) == MEM)
 	output_address (XEXP (x, 0));
       else if (GET_CODE (x) == CONST_INT)
+	fprintf (file, "%ld", INTVAL (x));
-	fprintf (file, HOST_WIDE_INT_PRINT_DEC, INTVAL (x));
       else
 	output_addr_const (file, x);
     }
